corehq/apps/reports/models.py

```diff
@@ -482,7 +482,7 @@ def get_report_content(self, lang, attach_excel=False):
         except Exception:
             pass
 
-        if self.report.is_deprecated:
+        if getattr(self.report, 'is_deprecated', False):
```

Comment:
this is probably fine, but would it be better to only check `is_deprecated` if the object is an instance of a class that has the attribute?